docs/release/release-tasks.md

```diff
@@ -367,6 +367,7 @@ upcoming code freezes etc based on the [release timeline (1.4 example)](./releas
 
 Information can be distributed via:
 * `sig-cluster-lifecycle` mailing list
+  * Note: Ensure that the communication email was received by the community. If unsure, check with the maintainers if the email was pending for approval and needs to be unblocked.
```

Comment:
I think only sig cluster lifecycle leadership has the rights.

@fabriziopandini ?

Reply:
confirmed, only a few sig leaders can do this operation (access rights for mailing lists are tricky, I know there was some work in progress but I'm not really up to date)

Reply:
As per our understand folks have to be just members of the mailing list. I would keep this simple and just say whoever wants to send mails should join the mailing list before.

Then they can send mails, and if they see that the mail doesn't show up on the mailing list they can ask sig leadership / Fabrizio to check if the mail has to be approved

Reply:
member in this case means subscribed at the mailing list, sorry for the confusion

Reply:
Makes sense to me. Updated.